clang/test/lab1/travin_maksim/test.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -ast-dump -ast-dump-filter test -load %llvmshlibdir/TravinAlwaysInlinePlugin%pluginext -add-plugin add-always-inline %s | FileCheck %s
+
+// CHECK: FunctionDecl {{0[xX][0-9a-fA-F]+ <.+test\.cpp:([0-9]+:[0-9]|[0-9]+), (line|col):([0-9]+:[0-9]|[0-9]+)> (line|col):([0-9]+:[0-9]|[0-9]+) test1 'void \(\)'}}
+void test1() {};
+
```

Comment:
Please check that Always inline attribute is added where it is expected to be done so.